diff -up ./file.c.orig ./file.c
--- ./file.c.orig
+++ ./file.c
@@ -1455,7 +1455,7 @@
 	if (zpos.len >= FILEPOS_BITS/BASEB) {
 		/* copy the lower FILEPOS_BITS of the ZVALUE */
 		memset(&tmp, 0, sizeof(tmp)); /* FILEPOS could be non-scalar */
-		memcpy(&tmp, zpos.v, MIN(sizeof(tmp), FILEPOS_LEN);
+		memcpy(&tmp, zpos.v, MIN(sizeof(tmp), FILEPOS_LEN));
 	} else {
 		/* copy what bits we can into the temp value */
 		memset(&tmp, 0, sizeof(tmp)); /* FILEPOS could be non-scalar */
