diff -up ./doh-cli.orig ./doh-cli
--- ./doh-cli.orig
+++ ./doh-cli
@@ -19,6 +19,11 @@
     "cira-family": "https://family.canadianshield.cira.ca/dns-query",
     "securedns": "https://doh.securedns.eu/dns-query",
     "securedns-ads": "https://ads-doh.securedns.eu/dns-query",
+    "iij":           "https://public.dns.iij.jp/dns-query",
+    "adguard":       "https://dns.adguard.com/dns-query",
+    "adguard-family": "https://dns-family.adguard.com/dns-query",
+    "opendns":       "https://doh.opendns.com/dns-query",
+    "opendns-family":"https://doh.familyshield.opendns.com/dns-query",
 }
 
 RR = ["A", "AAAA", "CNAME", "MX", "NS", "SOA", "SPF", "SRV", "TXT", "CAA", "DNSKEY", "DS"]
